plugins/org.jboss.tools.livereload.ui/src/org/jboss/tools/livereload/ui/internal/command/OpenInWebBrowserViaLiveReloadProxyEnablementTester.java

```diff
@@ -34,7 +35,11 @@ public boolean test(Object receiver, String property, Object[] args, Object expe
 			return false;
 		}
 		final IServer appServer = appModule.getServer();
-		return appServer != null && WSTUtils.isServerStarted(appServer);
+		IModule[] modules = appModule.getModule();
+		return appServer != null
+				&& appServer.getModuleState(appModule.getModule()) == IServer.STATE_STARTED
```

Comment:
When a web module is deployed within an EAR (try wildfly-javaee7-webapp-ear-archetype), on WF10, it's state is IServer.STATE_UNKNOWN, so the tester returns false here and it makes it impossible to open via livereload

![](http://content.screencast.com/users/fbricon/folders/Jing/media/e3e101a5-31c0-4889-9dc1-2a72ac64f1c9/00000441.png)